doc/YamlConfigFile.md

```diff
@@ -0,0 +1,298 @@
+# YAML configuration file
+
+This page describes the configuration file, which controls the behavior of the Windows service.
+
+You can find configuration file samples in the [examples](../examples) directory of the source code repository.
+Actual samples are also being published as a part of releases on GitHub and NuGet.
```

Comment:
```suggestion
Actual samples are also being published as part of releases on GitHub and NuGet.
```

/cc @oleg-nenashev

Reply:
I am also not a native speaker :( 
I think we will be doing a major cleanup for v3 anyway